public-ui/src/app/views/survey-view/survey-view.component.html

```diff
@@ -185,7 +185,8 @@ <h2 *ngIf="!loading" class="secondary-display">
               </div>
             </div>
           </div>
-  </div>
+        </div>
```

Comment:
This should actually be at the top. I am not sure how it got to the bottom. 
This should be below the survey name or this is not relevant anymore.

Reply:
![image](https://user-images.githubusercontent.com/860209/54142550-f5b77400-43f5-11e9-9351-16960837d1a2.png)

Reply:
thanks